src/plugins/plexil/protobuf_adapter.cpp

```diff
@@ -394,6 +396,19 @@ ProtobufCommPlexilAdapter::release_message(const std::string& msg_id)
 	}
 }
 
+std::shared_ptr<google::protobuf::Message>
+ProtobufCommPlexilAdapter::get_message(const std::string& msg_id)
+{
+	std::shared_ptr<google::protobuf::Message> m;
```

Comment:
why not call it msg or something? (and same for below use of `m`)

Reply:
It's used locally in a very limited context (8 lines including blank ones), making this not much of an issue. It is also to keep it in-sync with the other functions in the file. These, in turn, are based on [src/libs/protobuf_clips/communicator.cpp](https://github.com/fawkesrobotics/fawkes/blob/ca1b9faf8b4f2143e2e0160c763215c0bf1b248c/src/libs/protobuf_clips/communicator.cpp). Hence, I'd like to keep them as-is to make it easy to port, e.g., fixes among the files, if that becomes necessary.